packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -14,19 +14,27 @@ const chatMessageStyles: ComponentSlotStylesInput<
 > = {
   root: ({ props: p, variables: v }): ICSSInJSStyle => ({
     display: 'inline-block',
+    position: 'relative',
+
+    marginLeft: p.mine ? pxToRem(140) : 0,
+    marginRight: !p.mine ? pxToRem(140) : 0,
+    maxWidth: `calc(100% - ${pxToRem(140)})`,
```

Comment:
can we have this magic `140` (or `100`, or even better, the whole margin value, `pxToRem(140`)) value in one place only (instead of 4) to:
- decrease risk of regression if it gets changed
- decrease the number of changes if it becomes a variable

Reply:
yes, lets do that :)